packages/rspack-cli/tests/build/basic/entry.env.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (env) {
+	console.log(
```

Comment:
I don't suggest you to use stdout for testing purpose. Please use other solutions like `DefinePlugin`

Reply:
Let's get it merged. This looks ok to me.